commander/src/bootstrapping/commands/generator/enable.ts

```diff
@@ -63,9 +60,21 @@ export abstract class EnableCommand extends BaseGeneratorCommand {
 
 	async run(): Promise<void> {
 		const { args, flags } = await this.parse(EnableCommand);
+
 		const { address } = args as { address: string };
 		const { 'use-status-value': useStatusValue } = flags;
 
+		if (
+			!flags['use-status-value'] &&
+			[flags.height, flags['max-height-generated'], flags['max-height-prevoted']].some(
+				v => typeof v !== 'number',
+			)
+		) {
+			throw new Error(
+				'All of the following must be provided: --height, --max-height-generated, --max-height-prevoted',
+			);
+		}
+
```

Comment:
if you move the check few lines below after `let { ... } = flags` it would be nicer, and L84-86 can be removed

Reply:
Done!

Reply:
yes, also the defined variables in the `let` can now be used 😃